resources/home/dnanexus/utils/query.py

```
@@ -0,0 +1,487 @@
+            executor.submit(
+                dxpy.describe, file, fields={"name": True, "id": True}
+            ): file
+            for file in gcnv_input + gcnv_output
+        }
+
+        for future in concurrent.futures.as_completed(concurrent_jobs):
+            # access returned output as each is returned in any order
+            try:
+                data = future.result()
+                calling_files[data["name"]] = data["id"]
+            except Exception as exc:
```

Ruff (0.8.0)

Do not catch blind exception: `Exception`

(BLE001)